bin/sui-dev-newsletter-07.md

```diff
@@ -51,6 +51,9 @@ Join the Sui Overflow Hackathon and Win $5000 in Sui with BlockEden.xyz! - [http
   * Move CLI allows you to specify the number of iterations to run each test that uses generated values. Find more details in this PR: [https://github.com/MystenLabs/sui/pull/18071#pullrequestreview-2102658215](https://github.com/MystenLabs/sui/pull/18071#pullrequestreview-2102658215)
   * introspection on emitted events in tests is now allowed. Find more details in this PR: [https://github.com/MystenLabs/sui/pull/17699](https://github.com/MystenLabs/sui/pull/17699)
   * It is no longer needed a code with `assert!` thanks to clever errors. Find more details in this PR: [https://github.com/MystenLabs/sui/pull/17882](https://github.com/MystenLabs/sui/pull/17882)
+* JSON-RPC Subscriptions are deprecated as of [PR #18099](https://github.com/MystenLabs/sui/pull/18099). As of the next release they'll be disabled by default but node operators can opt back into enabling them via configuration (instructions can be found in the PR) although the implementation will be fully removed from the node software within another 2 release cycles.
+* The `InputObject` and `ChangedObject` transaction filters for the JSON-RPC method `suix_queryTransactionBlocks` are being deprecated for JSON-RPC served by a Fullnode during the next release. They, as well as the backing indexes for these filters, will remain enabled by default for the next few releases. A configuration option will be provided to allow for node operators to disable these filters and remove the expensive backing indexes allowing them to free up a large amount of disk space.
+
```

Comment:
I would mention this also in the newsletter intro, maybe a small paragraph or in a Breaking Changes Annoucement paragraph:
JSON RPC Subscriptions are deprecated in the next release. Read more below.

Alternatively, we should highlight this in some ways because IT IS very important to get across the ecosystem... This could be done in CRM though.